### ISSUE TITLE
More basic examples (eg. telnet echo server)

### ISSUE THREAD

**Issue by tomek14** [helios-io/helios#66]
Hi,

I started to play with Helios and wanted to create basic echo Tcp server that works with any telnet client. In examples there is a TcpReactorServer, but it only works with RawSocket exmple. When using windows telnet, I can connect to the server, but typing any character in telnet client doesn't gets echoed. How can i achive this functionaly using Helios? It would be cool to have more basic examples, becouse documenation is very, very poor and examples are good starting point.

**Comment by TheDeadlyDutchi** [comment]
I agree. I can't even find any other examples than those already there, and those don't help that much.

**Comment by saitolabs** [comment]
+1

**Comment by sant0ro** [comment]
+1
